src/script/components/calling/CallParticipantsListItem/CallParticipantStatusIcons.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Icon} from 'Components/Icon';
+import {Participant} from 'src/script/calling/Participant';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+
+import {ParticipantMicOnIcon} from '../ParticipantMicOnIcon';
+
+interface CallParticipantStatusIconsProps {
+  callParticipant: Participant;
+}
+
+export const CallParticipantStatusIcons = ({callParticipant}: CallParticipantStatusIconsProps) => {
+  const {sharesCamera, sharesScreen, isActivelySpeaking, isMuted} = useKoSubscribableChildren(callParticipant, [
+    'sharesCamera',
+    'sharesScreen',
+    'isActivelySpeaking',
+    'isMuted',
+  ]);
+
+  return (
+    <>
+      {sharesScreen && <Icon.Screenshare className="screenshare-icon" data-uie-name="status-screenshare" />}
```

Comment:
Icons are hidden from screen readers(I believe we have aria-hidden=true} set in the component, what's the alternative you are providing for screen readers?

Reply:
you can turn on screen reader and focus on the icon to listen the announcement

Reply:
This is refactoring only PR, we don't want to introduce new features with it. But it's definitely good point.